CHANGELOG.md

```diff
@@ -4,6 +4,17 @@
 
 [1]: https://pypi.org/project/google-cloud-bigquery/#history
 
+### [1.26.1](https://www.github.com/googleapis/python-bigquery/compare/v1.26.0...v1.26.1) (2020-07-23)
+
+### Documentation
+
+* Migrated additional code samples from
+  https://github.com/GoogleCloudPlatform/python-docs-samples
```

Comment:
You can link to the relevant PR, if you think it's necessary, but can also be done separately - shipping the bugfix has far more value for users.

Reply:
I considered it, but since it's also a history migration I figured I'd omit it.